.gitignore

```diff
@@ -103,6 +103,7 @@ celerybeat.pid
 
 # Environments
 .env
+!scripts/.env
```

Comment:
IIUC, this `.env` file contains configurations related to ingress service and not specific to ROS service so would it be good to rename it accordingly?